java/client/src/main/java/glide/api/models/commands/SortBaseOptions.java

```diff
@@ -0,0 +1,112 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * Optional arguments to {@link glide.api.commands.GenericBaseCommands#sort(String, SortOptions)},
+ * {@link glide.api.commands.GenericBaseCommands#sortReadOnly(String, SortOptions)}, {@link
+ * glide.api.commands.GenericBaseCommands#sortWithStore(String, String, SortOptions)}, {@link
+ * glide.api.commands.GenericCommands#sort(String, SortStandaloneOptions)}, {@link
+ * glide.api.commands.GenericCommands#sortReadOnly(String, SortStandaloneOptions)}, and {@link
+ * glide.api.commands.GenericCommands#sortWithStore(String, String, SortStandaloneOptions)}
+ *
+ * @see <a href="https://redis.io/commands/sort/">redis.io</a> and <a
+ *     href="https://redis.io/docs/latest/commands/sort_ro/">redis.io</a>
```

Comment:
This is unreadable - it is rendered as
> see `redis.io` and `redis.io`

I'd change it to
> see `sort` and `sort_ro` on redis.io

See https://github.com/aws/glide-for-redis/pull/1593/files#diff-79f8181a9c26e622050960edf8931a8b397cb1c0d5900c7a8a47f3e4ceae448bR25-R27
Feel free to reword & improve

Reply:
It will be inconsistent with all other Options files :(

Reply:
Need to update others too, I found that a day ago
You can keep it as is, will update links for all options later